Index: qtbase-opensource-src-5.3.2/src/plugins/platforms/eglfs/qeglfswindow.cpp
===================================================================
--- qtbase-opensource-src-5.3.2.orig/src/plugins/platforms/eglfs/qeglfswindow.cpp
+++ qtbase-opensource-src-5.3.2/src/plugins/platforms/eglfs/qeglfswindow.cpp
@@ -85,11 +85,13 @@
         if (isRaster() && screen->compositingWindow())
             return;
 
+#if 0
 #if !defined(Q_OS_ANDROID) || defined(Q_OS_ANDROID_NO_SDK)
         // We can have either a single OpenGL window or multiple raster windows.
         // Other combinations cannot work.
         qFatal("EGLFS: OpenGL windows cannot be mixed with others.");
 #endif
+#endif
 
         return;
     }
@@ -188,6 +190,9 @@
 {
     QRect rect;
     bool forceFullscreen = m_flags.testFlag(HasNativeWindow);
+
+    forceFullscreen = true;
+
     if (forceFullscreen)
         rect = screen()->availableGeometry();
     else
