Index: qtbase-opensource-src-5.4.0/src/plugins/platforms/eglfs/qeglfswindow.cpp
===================================================================
--- qtbase-opensource-src-5.4.0.orig/src/plugins/platforms/eglfs/qeglfswindow.cpp
+++ qtbase-opensource-src-5.4.0/src/plugins/platforms/eglfs/qeglfswindow.cpp
@@ -79,11 +79,13 @@
             return;
         }
 
+#if 0
 #if !defined(Q_OS_ANDROID) || defined(Q_OS_ANDROID_NO_SDK)
         // We can have either a single OpenGL window or multiple raster windows.
         // Other combinations cannot work.
         qFatal("EGLFS: OpenGL windows cannot be mixed with others.");
 #endif
+#endif
 
         return;
     }
@@ -183,6 +185,9 @@
 {
     QRect rect;
     bool forceFullscreen = m_flags.testFlag(HasNativeWindow);
+
+    forceFullscreen = true;
+
     if (forceFullscreen)
         rect = screen()->availableGeometry();
     else
